ui/src/app/pages/homepage/quick-tour-modal.tsx

```diff
@@ -331,7 +325,7 @@ export class QuickTourReact extends React.Component<QuickTourReactProps, QuickTo
       </div> :
       <React.Fragment>
         <div style={styles.modalBackdrop}/>
-        <div style={styles.mainStyling} data-test-id='quick-tour-react' className='quickTourReact'>
+        <div style={{ ...styles.mainStyling, display: 'flex', flexDirection: 'column'}} data-test-id='quick-tour-react' className='quickTourReact'>
```

Comment:
We can use FlexColumn tag here (which is basically a div with display: 'flex', flexDirection: 'column')

Reply:
Thanks - done.